### PR TITLE
MBST-14272: added Reviews/Ratings:

### PR DESCRIPTION
**Pull request by atish-metabroadcast**
* added fields to Described
* LegacyTransformer understands reviews and ratings
* Described serialiser understands ratings/reviews
* Described serialiser understands ratings/reviews for content_group

### CODE REVIEW

File: atlas-core/src/main/java/org/atlasapi/entity/Rating.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        Rating rating = (Rating) o;
+        return Float.compare(rating.value, value) == 0 &&
+                Objects.equals(type, rating.type) &&
+                publisher == rating.publisher;
+    }
+
+    @Override public int hashCode() {
```
Comment (Dalamar42) [inline atlas-core/src/main/java/org/atlasapi/entity/Rating.java]: Use the Java7 template for equals/hashCode please

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java [hunk 2]
```diff
@@ -121,4 +130,32 @@ protected Award transformAward(org.atlasapi.media.entity.Award awardLegacy) {
         return award;
     }
 
+    protected Iterable<Review> transformReviews(Iterable<org.atlasapi.media.entity.Review> legacyReviews) {
+        // Guava Optional used here because converting from Iterable to Stream to filter and collect
+        // is very cumbersome
+        Iterable<Optional<Review>> transformedReviewsGoodAndBad =
```
Comment (Dalamar42) [inline atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java]: Getting an iterable from a stream is cumbersome, but I don't think this is more readable. Also we are trying to move away from using Guava functional style for consistency and this is taking us back to it a bit

**Comment by Dalamar42** [comment]
OLGTM

File: atlas-core/src/main/java/org/atlasapi/entity/Rating.java (hunk 1 above)
Comment (atish-metabroadcast) [inline atlas-core/src/main/java/org/atlasapi/entity/Rating.java]: According to IJ, that is the Java7 template.

File: atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java (hunk 2 above)
Comment (atish-metabroadcast) [inline atlas-legacy/src/main/java/org/atlasapi/system/legacy/DescribedLegacyResourceTransformer.java]: Much easier cheating.  Changed the signature of transformReviews to a Collection.  Happy days.

**Pull request merged**